### DISCUSSION TITLE
How to make communications more effective

### DISCUSSION THREAD

**Discussion by AJLinn** [ArctosDB/arctos#7630] [General]
From #7280:
@Jegelewicz commented:
> It has become to clear to me over the last year that we are failing to communicate with the community even though we are spending a good amount of time in meetings, writing documentation and crafting newsletters that I am now confident that nobody is reading. What can we do to make sure that important information gets to the people who need it?

We might be looking at a case of information overload.

I, for one, feel like I"m drowning in Github notifications. I tried the option of only getting the messages if I'm mentioned, but then changes happen that negatively impact my collection because I didn't have time set aside to regularly monitor the issues closely, or got lost in the dozens of replies to an issue. I do try to read the newsletter, but again, there is so much information coming at us all day long, it's hard to recognize the important stuff for our collection. 

I'm finding that it's really easy to miss a string of communications originating in Github, even one with dozens of comments, when we're outside of the Arctos environment, working with students, installing an exhibition, teaching a class, hosting a researcher, or in on-site museum-based meetings. I think we're taking on SO MANY issues right now that have BIG impacts for many people, it's creating a lot of instability and distrust in our user base.

I know the goal is to be constantly improving... but our users want a system they know how to use, that is familiar, and does what we need it to do. For those hundreds of users who are not participating in the Github forum, or attending AWG meetings or issues meetings, or attending any of the many committee or casual drop in groups our Arctos team has available to us, all they know is that they log in on Monday and the interface looks and works one way. They log in on Thursday and things have changed.

Perhaps we can have a discussion at an upcoming meeting to brainstorm about how we can make our communications more effective? I don't have a solution but I think it's worth listening to people about their frustrations. UAM has created our own internal user group - I'd be willing to bring some of our group's concerns forward if they want me to.

**Comment by Jegelewicz** [comment]
Make better use of [Arctos Dev](https://github.com/ArctosDB/arctos-dev/issues) for communications.

**Comment by campmlc** [comment]
I found the demos at the working group today extremely helpful and effective. I haven't been able to attend to all the discussions and Github issues, but Ive been aware of them because of the email announcements. This summary and opportunity to preview prior to implementation during a working group meeting is great. I especially appreciate the recording of the demos which can be shared with colleagues. For changes of this magnitude, this route seems very appropriate.

**Comment by dustymc** [comment]
>Make better use of [Arctos Dev](https://github.com/ArctosDB/arctos-dev/issues) for communications.

I was just going to suggest killing that, I don't think anyone uses it...

I think we're doing a bit of a bunch of things, and WHATEVER we do someone shows up wishing we'd have done something else. I need guidance, one dev path (and sufficient support/resources if this turns into yet another complication).

Seems like perhaps keeping github for 'dev stuff' and using the arctos banner (new page, whatever) for 'public stuff' might have some promise?

EDIT: @campmlc chat from meeting:

>Communication directly through Arctos and via email is better than through GitHub.

**Comment by campmlc** [comment]
I also support having a preview or a link to a preview of the new form prior to implementation, as others remarked in today's meeting. Very helpful, and it may reach users who are not getting other forms of communication.

**Comment by dustymc** [comment]
Discussed post-meeting: I will continue to use arctos-dev to announce significant changes. Anyone can subscribe to notifications from there, and doing so should result in one notification for every significant update.

(I still struggle with the process leading up to posting things there, I'll keep interpreting silence as enthusiastic approval until someone proposes some other actionable pathway.)

Are there further action items or should this be closed?
